experiments/list_unsupported.sh

```diff
@@ -84,10 +84,18 @@ compile_error_occured=0
 for filename in $(find ${path} -name '*.adb'); do
    echo >&2 "Compiling $filename..."
    echo "---------- COMPILING: $filename" >>"$file_name".txt
-   "${GNAT2GOTO}" ${include_path} "${filename}" >>"$file_name".txt 2>&1
-   if [ "$?" -gt 0 ]; then
-      compile_error_occured=1
-      echo "---------- FAILED ----------------------------" >>"$file_name".txt
+   "${GNAT2GOTO}" ${include_path} "${filename}" > "$file_name".txt.compiling 2>&1
+   result=$?
+   cat "$file_name".txt.compiling >> "$file_name".txt
+   if [ "$result" -gt 0 ]; then
+      # Got a non-zero exit code, check where we managed to get a list of
```

Comment:
check where -> check if?

Reply:
Thanks - I meant to write "whether", but "check if" is better - fixed.